shared/src/commonMain/resources/MR/base/strings.xml

```diff
@@ -438,4 +438,6 @@
     <string name="streak_recovery_modal_warning">This choice cannot be undone</string>
     <string name="streak_recovery_modal_restore_streak">Restore streak</string>
     <string name="streak_recovery_modal_no_thanks">No, thanks</string>
+    <string name="streak_recovery_modal_success">Streak successfully recovered</string>
+    <string name="streak_recovery_modal_error">Failed to recover streak</string>
```

Comment:
cc @vladkash support this on iOS ^^

Reply:
@ivan-magda @XanderZhu 
We dispatch the same view actions `ShowNetworkRequestStatus` for statuses when user taps "Restore streak" and "No thanks", so I think it's wrong to show "Streak successfully recovered" when used tapped "No thanks". On iOS I just show status with ✅ and ❌

Reply:
@vladkash @XanderZhu I added string resources:
1. https://github.com/hyperskill/mobile-app/pull/526/commits/1d9278e7ca558234dd84618a5193daf746b85c00
2. https://github.com/hyperskill/mobile-app/pull/526/commits/fa34e870414a30a6fff51e9d9e98fec489144e3e